.github/workflows/build-docker-image.yml

```
@@ -13,27 +15,28 @@ jobs:
         os:
           - ubuntu
           - debian
+          - rhel
 
     steps:
       - name: Checkout
         uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)